tests/modules/individuals/conftest.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=missing-docstring,redefined-outer-name
+import pytest
+
+from app.modules.encounters.models import Encounter
+from app.modules.individuals.models import Individual
+from app.modules.users.models import User
+
+
+def _user():
```

Comment:
Seems to be a duplication of https://github.com/WildMeOrg/houston/blob/master/tests/utils.py#L148